README.rst

```diff
@@ -1,14 +1,17 @@
 nested_lookup
 #############
 
-.. image:: https://img.shields.io/badge/pypi-0.2.01-green.svg
+.. image:: https://img.shields.io/badge/pypi-0.3.0-green.svg
   :target: https://pypi.python.org/pypi/nested-lookup
 
-A small Python library which enables:
+Python library which enables:
 
-#. key lookups on deeply nested documents.
-#. fetching all keys from a nested dictionary.
-#. get the number of occurrences of a key/value from a nested dictionary
+#. (nested_lookup) key lookups on deeply nested documents.
```

Comment:
Please include: "The `nested_lookup` package provides many Python functions for working with deeply nested documents. A document in this case is a a mixture of Python `dictionary` and list` objects typically derived from YAML or JSON"

Reply:
Please reword this to explain that the `nested_lookup` function will return a `list` of all values which match the given `key` in the given `document`

Reply:
I would prefer if we break this ordered list into more of a "reference", like:

```
*nested_lookup:*
    Perform a key lookup on a deeply nested document. Returns all matches in a `list`.
```